cloudferrylib/os/identity/keystone.py

```diff
@@ -32,6 +33,7 @@
 
 LOG = log.getLogger(__name__)
 NO_TENANT = ''
+RETRY_COUNT = 5
```

Comment:
No, please do not use the global variable. 
Take a look at any other modules and find how we are using the CONF variable